inc/fw-update.php

```diff
@@ -201,13 +201,25 @@ function spbc_security_firewall_update__get_multifiles()
             $tries_for_download_again         = 3 + (int)(count($result['file_urls']) / 20);
             $spbc->fw_stats['files_count']    = count($result['file_urls']);
             $spbc->fw_stats['update_percent'] = 10;
+            $file_urls = array();
+            $file_hashes = array();
 
             $spbc->save('fw_stats', true, false);
 
+            // Create array with urls
+            foreach ($result['file_urls'] as $file_url_info) {
+                $file_urls[] = isset($file_url_info[0]) ? $file_url_info[0] : '';
+                $file_hashes[] = isset($file_url_info[1]) ? $file_url_info[1] : '';
+            }
+
+            // Save info about urls and hashes
+            $spbc->settings['secfw_data_files_info'] = $file_hashes;
```

Comment:
Save this metadata to the other storage, not in the settings storage.

Reply:
DONE